lib/puppet/configurer.rb

```diff
@@ -157,12 +173,17 @@ def run_internal(options)
       # We only need to find out the environment to run in if we don't already have a catalog
       unless options[:catalog]
         begin
-          if node = Puppet::Node.indirection.find(Puppet[:node_name_value],
-              :environment => Puppet::Node::Environment.remote(@environment),
-              :configured_environment => Puppet[:environment],
-              :ignore_cache => true,
-              :transaction_uuid => @transaction_uuid,
-              :fail_on_404 => true)
+          node = nil
+          nodeRetrTime = thinmark do
```

Comment:
Nit picky comment: the preferred naming convention for local variables is to use `snake_case`.